Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -79,6 +79,27 @@ open class ReadingThemesControlsViewController: UIViewController, AnalyticsConte
         NSObject.cancelPreviousPerformRequests(withTarget: self)
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        let tap = UITapGestureRecognizer(target: self, action: #selector(self.handleTap(_:)))
+        slider.addGestureRecognizer(tap)
+    }
+    
+    func handleTap(_ sender: UIGestureRecognizer) {
+        let pointTapped: CGPoint = sender.location(in: self.view)
+        
+        let positionOfSlider: CGPoint = slider.frame.origin
+        let widthOfSlider: CGFloat = slider.frame.size.width
+        let newValue = ((pointTapped.x - positionOfSlider.x) * CGFloat(slider.numberOfSteps) / widthOfSlider)
```

Comment:
you could also remove the position in slider adjustment here since the point returned by `location(in: slider)` should have an x coordinate between 0 and `slider.bounds.size.width` - make sure to test trying to tap outside the slider bounds though